extensions/import/src/wizard/flatFileWizard.ts

```diff
@@ -22,6 +22,7 @@ export class FlatFileWizard {
 	public page2: azdata.window.WizardPage;
 	public page3: azdata.window.WizardPage;
 	public page4: azdata.window.WizardPage;
+	public createDerivedColumnButton: azdata.window.Button;
```

Comment:
NIT: can you group this with the importAnotherFileButton declaration and add a space between line 24 and line 25

Reply:
Can you make this button private and add a function like 
https://github.com/microsoft/azuredatastudio/blob/a92ba424ac0fdfda1fa0e10d0b4a89d76af184b7/extensions/import/src/wizard/flatFileWizard.ts